cmd/testscript/help.go

```diff
@@ -36,6 +36,12 @@ where VAR is the name of the variable. Variables override testscript-defined
 values, with the exception of WORK which cannot be overridden. The -e flag can
 appear multiple times to specify multiple variables.
 
+The -update flag specifies that if a cmp command within a testscript fails and
+its second argument refers to a file inside the testscript file, the command
+will succeed and the testscript file will be updated to reflect the actual
+content. As such, this is the cmd/tescript equivalent of
```

Comment:
```suggestion
content. As such, this is the cmd/testscript equivalent of
```

Reply:
Thanks for the catch. Just as I was writing this I was getting frustrated that for some reason my editor's spell checker does not seem to want to work within string literals.